mobile/components/ThreadItem/style.js

```diff
@@ -53,11 +54,24 @@ export const FacepileContainer = styled.View`
 export const EmptyParticipantHead = styled.Text`
   color: ${props => props.theme.text.alt};
   background: ${props => props.theme.bg.wash};
-  border-radius: 15px;
+  border-radius: ${props => (props.radius ? `${props.radius}px` : '15px')};
   text-align: center;
   text-align-vertical: center;
   font-size: 12px;
   font-weight: 600;
-  width: 30px;
+  height: ${props => (props.size ? `${props.size}px` : '30px')};
+  width: ${props => (props.size ? `${props.size}px` : '30px')};
   overflow: hidden;
 `;
+
+const stackizeHeads = (
+  component,
+  { marginRight = '-10px', borderWidth = '2px', borderColor = '#FFFFFF' } = {}
+) => styled(component)`
+  margin-right: ${marginRight};
+  border-width: ${borderWidth};
+  border-color: ${borderColor};
+`;
+
+export const StackedEmptyParticipantHead = stackizeHeads(EmptyParticipantHead);
+export const StackedAvatar = stackizeHeads(Avatar);
```

Comment:
We should use the `css` helper here instead of wrapping it in another component:

```JS
import { css } from 'styled-components';

const stackingStyles = css`
  margin-right: -10px;
  border-width: 2px;
  border-color: #FFFFFF;
`

export const StackedEmptyParticipantHead = styled(EmptyParticipantHead)`
  ${stackingStyles}
`

export const StackedAvatar = styled(Avatar)`
  ${stackingStyles}
`
```

Reply:
Will refactor this :+1: thanks! I went with an HoC first as I did not know what was the preferred way of implementing this behavior.

Reply:
Will make this change :+1:. Should I keep the HoC, or it is overkill for this? @mxstbr

Reply:
No need for the HoC